tests-upgrade/tests-emitter/Sphere.Management/target/README.md

```diff
@@ -0,0 +1,24 @@
+<!-- region Generated -->
```

Comment:
does target folder need to be checked-in?

Reply:
Yes, it is manually maintained.

Reply:
> Yes, it is manually maintained.

isn't it output of tsp compile --emit ? why the generated content need to be checked-in?

Reply:
The output of tsp compile will be put in **generated** folder. **Target** folder contains the expected code.